src/averageAges.js

```diff
@@ -14,12 +14,34 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
   // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
```

Comment:
Part of this line has code that is repeated. Could you possibly turn it into a variable or a global variable?